Apps/Playground/CMakeLists.txt

```diff
@@ -49,10 +64,12 @@ if(APPLE)
         set_source_files_properties(${SCRIPTS} ${BABYLON_SCRIPTS} ${DEPENDENCIES} PROPERTIES MACOSX_PACKAGE_LOCATION "Resources/Scripts")
         set_source_files_properties(${REFERENCE_IMAGES} PROPERTIES MACOSX_PACKAGE_LOCATION "Resources/ReferenceImages")
     endif()
-    set(ADDITIONAL_LIBRARIES ${ADDITIONAL_LIBRARIES}
-        PRIVATE ${JAVASCRIPTCORE_LIBRARY}
-        PRIVATE NativeCamera)
+    set(ADDITIONAL_LIBRARIES ${ADDITIONAL_LIBRARIES} PRIVATE ${JAVASCRIPTCORE_LIBRARY})
     set(RESOURCE_FILES ${STORYBOARD})
+
+    # Remove -Wall and -Werror from the debug flags to make simulators work.
+    string(REPLACE "-Wall" "" CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG}")
+    string(REPLACE "-Werror" "" CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG}")
```

Comment:
What are the warnings/errors that make things otherwise not work for simualtors?

Reply:
It was treating internal warnings as errors, like this:

![CleanShot 2024-08-22 at 17 11 52@2x](https://github.com/user-attachments/assets/34071e90-7124-434d-b032-bdc6c38169c5)